tasks/release.py

```diff
@@ -117,3 +121,139 @@ def _find_v6_tag(ctx, v7_tag):
+    if highest_version["rc"] is not 0:
+        version = "{}-rc.{}".format(version, highest_version["rc"])
+    return version
+
+
+@task
+def create_rc(ctx, omnibus_ruby_version = "datadog-5.5.0"):
+
+    """
+    Creates new entry in the release.json file for a new RC.
+    Looks at the current release.json to find the highest version released and create a new RC based on that.
+    """
+
+    github_token = os.environ.get('GITHUB_TOKEN')
+    if github_token is None:
+        print(
+            "Error: set the GITHUB_TOKEN environment variable.\nYou can create one by going to"
+            " https://github.com/settings/tokens. It should have at least the 'repo' permissions.")
+        return Exit(code=1)
+
+    version_re = re.compile('(\\d+)[.](\\d+)[.](\\d+)(-rc\\.(\\d+))?')
+
+    auth = "https://{}:x-oauth-basic@".format(github_token)
+
+    with open("release.json", "r") as release_json_stream:
+        release_json = json.load(release_json_stream)
+
+    highest_version = None
+    highest_jmxfetch_version = None
+
+    for key, value in release_json.items():
+        match = version_re.match(key)
+        if match:
+            this_version = _create_version_dict_from_match(match)
+            if _is_version_higher(this_version, highest_version):
+                highest_version = this_version
+                match = version_re.match(value["JMXFETCH_VERSION"])
+                if match:
+                    highest_jmxfetch_version = _create_version_dict_from_match(match)
+
+    highest_version["rc"] = highest_version["rc"] + 1
+    new_rc = _stringify_version(highest_version)
+    print("Creating {}".format(new_rc))
+
+    integration_rc = _get_repo_rc_version(auth, "integrations-core", highest_version, version_re)
+    print("Integration-Core's highest tag is {}".format(integration_rc))
+
+    omnibus_software_rc = _get_repo_rc_version(auth, "omnibus-software", highest_version, version_re)
+    print("Omnibus-Software's highest tag is {}".format(omnibus_software_rc))
+
+    jmxfetch_rc = _get_repo_rc_version(auth, "jmxfetch", highest_jmxfetch_version, version_re)
+    print("Jmxfetch's highest tag is {}".format(jmxfetch_rc))
+
+    jmxfetch = urllib.urlopen("https://bintray.com/datadog/datadog-maven/download_file?file_path=com%2Fdatadoghq%2Fjmxfetch%2F{}%2Fjmxfetch-{}-jar-with-dependencies.jar")
+    jmxfetch_sha256 = hashlib.sha256(jmxfetch.read()).hexdigest()
+
+    print("Jmxfetch's SHA256 is {}".format(jmxfetch_sha256))
+
+    new_rc_config = {
+        "INTEGRATIONS_CORE_VERSION": integration_rc,
+        "OMNIBUS_SOFTWARE_VERSION": omnibus_software_rc,
+        "OMNIBUS_RUBY_VERSION": omnibus_ruby_version,
+        "JMXFETCH_VERSION": jmxfetch_rc,
+        "JMXFETCH_HASH": jmxfetch_sha256
+    }
+
+    release_json[new_rc] = new_rc_config
+
+    # We need to release Agent 6 as well
+    highest_version["major"] = 6
+    new_rc = _stringify_version(highest_version)
+    release_json[new_rc] = new_rc_config
```

Comment:
I'd put this behind a config flag (defaulting to true), to give us the option to differentiate Agent 7 and Agent 6 releases in the future.

Reply:
I'd see something like a `majors` parameter where you specify a list of major versions to release, with `7,6` as the default, which we parse as a list at the start of the function. Then you do the full work for the first specified version and loop these 3 lines for the following versions.